OrbitGl/PresetsDataView.cpp

```diff
@@ -153,13 +199,45 @@ void PresetsDataView::DoFilter() {
 
 void PresetsDataView::OnDataChanged() {
   indices_.resize(presets_.size());
+  modules_.resize(presets_.size());
   for (size_t i = 0; i < presets_.size(); ++i) {
     indices_[i] = i;
+    std::vector<ModuleView> modules;
+    for (const auto& pair : presets_[i]->preset_info().path_to_module()) {
+      modules.push_back(
+          ModuleView(Path::GetFileName(pair.first), pair.second.function_hashes_size()));
+    }
+    modules_[i] = modules;
```

Comment:
std::move

Reply:
EDIT: You could also remove the temporary variable if you want. But I'm also fine with adding move.

Reply:
Added std::move